errors/es-mx.po

```diff
@@ -0,0 +1,1474 @@
+msgid "Format error in certificate's notBefore field"
+msgstr "Error de formato en el campo notBefore del certificado"
+
+#: templates/ERR_FORWARDING_DENIED+html.body.div.blockquote.p:24-1
+msgid "Forwarding Denied."
+msgstr "Reenvío denegado."
+
+#: templates/ERR_GATEWAY_FAILURE+html.body.div.blockquote.p:24-1
+msgid "Gateway Proxy Failure"
+msgstr "Error en puerta de enlace del proxy"
+
+#: templates/ERR_ACCESS_DENIED+html.body.div.p:35-1
+#: templates/ERR_ACL_TIME_QUOTA_EXCEEDED+html.body.div.p:36-1
+#: templates/ERR_AGENT_CONFIGURE+html.body.div.p:57-1
+#: templates/ERR_AGENT_WPAD+html.body.div.p:57-1
+#: templates/ERR_CACHE_ACCESS_DENIED+html.body.div.p:36-1
+#: templates/ERR_CACHE_MGR_ACCESS_DENIED+html.body.div.p:36-1
+#: templates/ERR_CANNOT_FORWARD+html.body.div.p:43-1
+#: templates/ERR_CONFLICT_HOST+html.body.div.p:41-1
+#: templates/ERR_CONNECT_FAIL+html.body.div.p:38-1
+#: templates/ERR_DIR_LISTING+html.body.div.p:39-1
+#: templates/ERR_DNS_FAIL+html.body.div.p:40-1
+#: templates/ERR_ESI+html.body.div.p:40-1
+#: templates/ERR_FORWARDING_DENIED+html.body.div.p:36-1
+#: templates/ERR_FTP_DISABLED+html.body.div.p:36-1
+#: templates/ERR_FTP_FAILURE+html.body.div.p:40-1
+#: templates/ERR_FTP_FORBIDDEN+html.body.div.p:40-1
+#: templates/ERR_FTP_NOT_FOUND+html.body.div.p:42-1
+#: templates/ERR_FTP_PUT_CREATED+html.body.div.p:24-1
+#: templates/ERR_FTP_PUT_ERROR+html.body.div.p:41-1
+#: templates/ERR_FTP_PUT_MODIFIED+html.body.div.p:24-1
+#: templates/ERR_FTP_UNAVAILABLE+html.body.div.p:41-1
+#: templates/ERR_GATEWAY_FAILURE+html.body.div.p:37-1
+#: templates/ERR_ICAP_FAILURE+html.body.div.p:42-1
+#: templates/ERR_INVALID_REQ+html.body.div.p:50-1
+#: templates/ERR_INVALID_RESP+html.body.div.p:37-1
+#: templates/ERR_INVALID_URL+html.body.div.p:43-1
+#: templates/ERR_LIFETIME_EXP+html.body.div.p:35-1
+#: templates/ERR_NO_RELAY+html.body.div.p:35-1
+#: templates/ERR_ONLY_IF_CACHED_MISS+html.body.div.p:35-1
+#: templates/ERR_PRECONDITION_FAILED+html.body.div.p:37-1
+#: templates/ERR_PROTOCOL_UNKNOWN+html.body.div.p:35-1
+#: templates/ERR_READ_ERROR+html.body.div.p:37-1
+#: templates/ERR_READ_TIMEOUT+html.body.div.p:37-1
+#: templates/ERR_SECURE_CONNECT_FAIL+html.body.div.p:43-1
+#: templates/ERR_SHUTTING_DOWN+html.body.div.p:31-1
+#: templates/ERR_SOCKET_FAILURE+html.body.div.p:37-1
+#: templates/ERR_TOO_BIG+html.body.div.p:37-1
+#: templates/ERR_UNSUP_HTTPVERSION+html.body.div.p:35-1
+#: templates/ERR_UNSUP_REQ+html.body.div.p:35-1
+#: templates/ERR_URN_RESOLVE+html.body.div.p:35-1
+#: templates/ERR_WRITE_ERROR+html.body.div.p:37-1
+#: templates/ERR_ZERO_SIZE_OBJECT+html.body.div.p:35-1
+msgid "Generated %T by %h (%s)"
+msgstr "Generado %T por %h (%s)"
+
+#: templates/ERR_INVALID_REQ+html.body.div.ul.li.p:35-5
+msgid ""
+"HTTP/1.1 <q>Expect:</q> feature is being asked from an HTTP/1.0 software."
+msgstr ""
+"HTTP/1.1 <q>Esperado:</q> la característica ha sido requerida desde un "
```

Comment:
"Expect:" is an HTTP header name from the protocol syntax. It needs to be left un-translated.

For reader understanding it could be accompanied by the translated meaning something like this:
```suggestion
"HTTP/1.1 esperado (<q>Expect:</q>) la característica ha sido requerida desde un "
```